ir/helper.go

```diff
@@ -335,16 +335,19 @@ type ReturnAttribute interface {
 	IsReturnAttribute()
 }
 
+// TODO: figure out definition of UnwindTarget.
+
 // UnwindTarget is an unwind target.
 //
 // An UnwindTarget has one of the following underlying types.
 //
 //    *ir.Block
 //    ir.UnwindToCaller
 type UnwindTarget interface {
+	value.Value
 	// isUnwindTarget ensures that only unwind targets can be assigned to the
 	// ir.UnwindTarget interface.
-	isUnwindTarget()
+	//isUnwindTarget()
```

Comment:
Why we would need to comment out this line? If we have a good reason to remove it then just remove it I guess.

Reply:
> Why we would need to comment out this line? If we have a good reason to remove it then just remove it I guess.

I removed this now. I left it uncommented as I wasn't entirely sure how to handle the UnwindTarget sumtype. The general issue tracking how to handle sumtypes is #59.

Reply:
I still see the commented line here in the files changed view?